src/sentry/features/temporary.py

```diff
@@ -267,8 +267,6 @@ def register_temporary_features(manager: FeatureManager):
     manager.add("organizations:performance-use-metrics", OrganizationFeature, FeatureHandlerStrategy.INTERNAL, api_expose=True)
     # Enable standalone cls and lcp in the web vitals module
     manager.add("organizations:performance-vitals-standalone-cls-lcp", OrganizationFeature, FeatureHandlerStrategy.FLAGPOLE, api_expose=True)
-    # Enable default explore queries in the new side nav
-    manager.add("organizations:performance-default-explore-queries", OrganizationFeature, FeatureHandlerStrategy.FLAGPOLE, api_expose=True)
```

Comment:
hmm, could be wrong here but removing the feature along with it's usage might be problematic. 
You'd definitely want to remove any frontend checks first. After that, if you merged this first I think the options automator would complain that a flag isn't registered, but if you merged the options automator Pr first then some users wouldn't have the feature for a moment.

Instead maybe do things in the following order?
1. remove flag usage from frontend
2. remove flag usage from backend
3. remove entry in options automator
4. remove entry from temporary.py

Reply:
You're definitely right about removing usage from the frontend first. I _think_ we've been able to remove backend usage and the flag from `temporary.py` in the same PR in the past, but it doesn't hurt to separate them. I'll take `temporary.py` changes out and into a separate PR just to be safe!

Reply:
Also fwiw, I think the options Automator complaining won't actually have any side effects, but better to be safe!